src/flagser_bindings.cpp

```diff
@@ -118,20 +107,23 @@ PYBIND11_MODULE(flagser_pybind, m) {
       }
     }
 
+    // If approximation is negative it falls back to type::numeric_limits
+    size_t max_entries =
+        approximation >= 0 ? approximation : std::numeric_limits<size_t>::max();
+
+    // Disable cout
     std::cout.rdbuf(nullptr);
 
-    auto ret = compute_homology(graph, named_arguments, max_entries, modulus);
+    // Running flagser's compute_homology routine
+    auto output = compute_homology(graph, named_arguments, max_entries, modulus);
```

Comment:
@gtauzin I like `persistence_computer`, but why not `persistence_computer_graphs` ? As I understood so far, `flagser` could return if we ask it a separate output for each component. Even though we don't use it at the moment, it could be the case in the future.

But I think that in the meantime we could go with `persistence_computer` till the day we would return something different.

Reply:
Maybe `subgraph_persistence_computer` as `persistence_computer_graph` gives me the feeling that we return a type of graph?